### DISCUSSION TITLE
Klipper / Voron / Fluidd / Mainsail - 3D Printer Software

### DISCUSSION THREAD

**Discussion by InAmberClad** [selfhst/icons#70] [New Icon Request]
Hi,

Can the following 3D printer logos please be added.

Thank you for the great work.

[Klipper SVG](https://github.com/fluidd-core/fluidd/blob/develop/public/logo_klipper.svg)

[Voron SVG](https://github.com/fluidd-core/fluidd/blob/develop/public/logo_voron.svg)

[Fluidd SVG](https://github.com/fluidd-core/fluidd/blob/develop/public/logo_fluidd.svg)

[Mainsail SVG](https://github.com/mainsail-crew/mainsail/blob/develop/public/img/logo.svg)

**Comment by shollyethan** [comment]
These have been added - thanks for the suggestion!
